app/models/form/lettings/questions/age.rb

```diff
@@ -11,5 +11,6 @@ def initialize(id, hsh, page, person_index:)
     @max = 120
     @min = 0
     @step = 1
+    @question_number = 31 + 4 * person_index
```

Comment:
i think i would of added brackets here so its obvious where the multiplication happens

Reply:
sure, good idea!